doc/snmp/snmp-configdb-migration-hld.md

```diff
@@ -31,7 +32,7 @@ SNMP|LOCATION
 SNMP|CONTACT
 
 admin@switch1:~$ redis-cli -n 4 hgetall "SNMP|LOCATION"
-1) "LOCATION"
+1) "Location"
```

Comment:
>Location [](http://example.com/codeflow?start=4&length=8)

Other field name are UPCASE. Do you want to align?

Reply:
I did this on purpose per what Joe asked me to do.  Also this aligns with what I've done in the commands for the other PR.